plugins/de.ovgu.featureide.ui/src/de/ovgu/featureide/ui/wizards/NewFeatureProjectPage.java

```diff
@@ -153,7 +154,15 @@ public void modifyText(ModifyEvent e) {
 				composerExtension = extensions[toolCB.getSelectionIndex()];
 			}
 		});
-		toolCB.select(0);
+		// set Feature Modelling as default composer
+		int featureModellingComposerIndex = 0;
+		for (int i = 0; i < extensions.length; i++) {
+			if (extensions[i].getName().equals("Feature Modeling")) {
```

Comment:
As changing the name of the composer anywhere else would cause this function to fail, I think adding a constant to the [Stringtable](https://github.com/FeatureIDE/FeatureIDE/blob/develop/plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/localization/StringTable.java) would be sensible.

Reply:
Your requested changes have been incorporated.

Reply:
Looks good, Thanks a lot!